searchcore/src/vespa/searchcore/proton/server/bucketmovejob.cpp

```diff
@@ -125,14 +125,15 @@ BucketMoveJob::moveDocuments(DocumentBucketMover &mover,
         bucketGuard = _frozenBuckets.acquireExclusiveBucket(mover.getBucket());
         if (! bucketGuard) {
             maybeDelayMover(mover, mover.getBucket());
-            return;
+            return true;
         }
     }
     assert(mover.getBucket() == bucketGuard->getBucket());
-    mover.moveDocuments(maxDocsToMove);
+    if ( ! mover.moveDocuments(maxDocsToMove)) return false;
```

Comment:
Consider adding braces to fix style here.

Reply:
Fixed